src/nupic/bindings/algorithms.i

```diff
@@ -101,9 +101,20 @@ _ALGORITHMS = _algorithms
 #include <nupic/proto/ConnectionsProto.capnp.h>
 #include <nupic/proto/SpatialPoolerProto.capnp.h>
 #include <nupic/proto/TemporalMemoryProto.capnp.h>
+%}
 
+//
+// Numpy API
+//
+%{
 #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
 #include <numpy/arrayobject.h>
+%}
+%init %{
+  import_array();
```

Comment:
math.i is imported previously in this file and it has this line:
https://github.com/mrcslws/nupic.core/blob/48622b48395a90ac821b82d8aed9e8a59d3e8b25/src/nupic/bindings/math.i#L90

That is probably why the line wasn't needed in this file previously. I've looked at the .i file relationships and we could probably simply the organization. I am open to any proposals for this.